catalog.json

```diff
@@ -40,12 +40,24 @@
       "type": "application/json",
       "title": "Products"
     },
+    {
+      "rel": "child",
+      "href": "./workflows/catalog.json",
+      "type": "application/json",
+      "title": "Workflows"
+    },
+    {
+      "rel": "child",
+      "href": "./experiments/catalog.json",
+      "type": "application/json",
+      "title": "Experiments"
+    },
     {
       "rel": "self",
       "href": "https://esa-earthcode.github.io/open-science-catalog-metadata/catalog.json",
       "type": "application/json"
     }
   ],
-  "updated": "2024-09-12T20:32:23.037110Z",
+  "updated": "2025-01-29T18:00:00.000000Z",
```

Comment:
```suggestion
  "updated": "2025-02-20T11:00:00.000000Z",
```

Reply:
I suppose git-clerk should do that. Either this or we remove it I'd say. Cc @silvester-pari